src/accessory/BUILD.gn

```diff
@@ -14,32 +14,31 @@
 
 import("//build_overrides/chip.gni")
 
-config("includes") {
+config("accessory_includes") {
```

Comment:
If users of the headers in this directory included them with "#include <accessory/..." this wouldn't be needed. And it would be more aligned with other directories from `src/`.